guide/ops/troubleshooting/web-console-issues.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: website-normal
+title: Web Console Issues
+toc: /guide/toc.json
+---
+
+## Page Does Not Load in Chrome, Saying ""Waiting for available socket..."
```

Comment:
I'm not sure if Karaf has anything to do with it, except that maybe folk have additional WARs running in Karaf that use web-sockets so will consume a socket.

I think the main issue (for me at least) is the login dialog being open in a different tab.